index.html

```diff
@@ -8,7 +8,7 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./css/style.css">
-    <title>Welcome to my Portfolio</title>
+    <title>my Portfolio Homepage</title>

```

Comment:
- [ ] To make your site more accessible, please make the page title adequately and briefly describes the content of the page. For example `**Mark's portifolio**,  **Mark full-stack webdeveloper**. Please kindly fix that.
-
![youmi](https://user-images.githubusercontent.com/65852621/139857507-e398d12a-7959-4d57-b27a-7ee0ac40b875.png)

Reply:
done thanks